cypress/support/pages/room_board/pageRoomBoards.js

```diff
@@ -7,9 +7,54 @@ class RoomBoards {
 	static #mainPageArea = '[id="main-content"]';
 	static #roomBoardTitleOnPage = '[data-testid="board-title"]';
 	static #boardMenuIcon = '[data-testid="board-menu-icon"]';
+	static #btnBoardMenuActionRename = '[data-testid="kebab-menu-action-rename"]';
 	static #dialogAddMultiColumnBoard = '[data-testid="dialog-add-multi-column-board"]';
 	static #dialogAddSingleColumnBoard = '[data-testid="dialog-add-single-column-board"]';
 	static #boardLayoutDialogTitle = '[data-testid="board-layout-dialog-title"]';
+	static #breadCrumbToRoomNavigationFromBoard = '[data-testid="breadcrumb-1"]';
+	static #multicolumnBoardSelector = '[data-testid="board-tile-subtitle-0"]';
+	static #copyOptionSelector = '[data-testid="kebab-menu-action-copy"]';
+	static #chipDraftSelector = '[data-testid="board-draft-chip"]';
+	static #publishMenuSelector = '[data-testid="kebab-menu-action-publish"]';
+	static #singleColumnBoardSelector = '[data-testid="board-tile-title-1"]';
+	static #multiColumnCopiedBoardSelector = '[data-testid="board-tile-title-2"]';
+	static #singleColumnCopiedBoardSelector = '[data-testid="board-tile-title-3"]';
+
+	verifyMultiColumnCopiedBoardVCardVisibleOnRoomDetailPage() {
+		cy.get(RoomBoards.#multiColumnCopiedBoardSelector).should("be.visible");
+	}
+
+	verifyListColumnCopiedBoardVCardVisibleOnRoomDetailPage() {
```

Comment:
same comment as above, including make it similar in all instances in the code either its `ListColumn` or `SingleColumn`

Reply:
done!